### DISCUSSION TITLE
Remove 'Abs' prefix from class names

### DISCUSSION THREAD

**Discussion by k9withabone** [audiobookshelf-flutter/audiobookshelf_api#2] [Q&A] [answered]
Should the 'Abs' prefix be removed from the class names? It made sense to have them when this was part of an app with multiple audiobook sources. Now though, it seems overly cumbersome.

**Comment by keaganhilliard** [comment, marked as answer]
It was probably overly cumbersome to write it that way in the first place even when I had multiple services. Dart lets you namespace imports so it seems a little pointless.

**Comment by k9withabone** [comment]
The whole package has been overhauled since I asked about this, so it's OBE now. I'll lock this conversation.
